lib/sqlalchemy/orm/strategy_options.py

```diff
@@ -1871,7 +1871,7 @@ def _reconcile(
             return replacement
 
         raise sa_exc.InvalidRequestError(
-            f"Loader strategies for {replacement.path} conflict"
+            f"Loader strategy replacement {replacement.path} is in conflict with existing strategy {existing.path}"
```

Comment:
this is fine though I think you need to install pre-commit and have it run because this will fail pep8 for line length

Reply:
Thanks for the hint. I indeed forgot to lint my changes.

Note that pre-commit did not work for me because of a version conflict with black:
```
stderr:
    ERROR: Package 'black' requires a different Python: 3.8.19 not in '>=3.9'
Check the log at /home/jan/.cache/pre-commit/pre-commit.log
```

I used `tox -e lint` instead.